infrastructure/eth-networks/private-testnet/README.md

```diff
@@ -0,0 +1,29 @@
+# Keep Network Private Testnet
+
+We set up a Keep Network Private Testnet that is accessible by the permitted parties.
+Here we hold the code helping us to create a bundles for the parties.
+
+The network runs against the link:https://goerli.net/[Ethereum Görli Testnet].
+
+The generated bundles contains a preinitialized Ethereum Account details. The account
+receives a stake delegation to the `provider` account with authorized `beacon` and
+`tbtc` application. This is a production-like experience for the Staking Providers,
+where they will receive stakes from their customers.
+
+## Scripts
+
+### Create New Bundle
```

Comment:
We should note that this script requires `asciidoctor` to be installed (`brew install asciidoctor`) and that `geth` needs to be on the `$PATH`. I failed both requirements while testing. 😉

Reply:
... and Docker!

Reply:
4d996cdd28f87d95c0377a664a6f4c2a4e10681d